plugins/linux/ifplugin/descriptor/interface.go

```diff
@@ -261,6 +271,24 @@ func (d *InterfaceDescriptor) Validate(key string, linuxIf *interfaces.Interface
 			return kvs.NewInvalidValueError(ErrVETHWithoutPeer, "peer_if_name")
 		}
 	}
+
+	if linuxIf.Type == interfaces.Interface_LOOPBACK {
+		nsCtx := nslinuxcalls.NewNamespaceMgmtCtx()
```

Comment:
I just realized that inside the scheduling algorithm, configuration is validated before the dependencies are checked (it was really just meant for the validation of the configuration, not the environment). For loopback configuration with microservice namespace, it is a valid case if the namespace doesn't exist yet (it will be pending and created when the notifications about microservice arrives). Therefore it may be better to put these checks into Create (and mark the "already-exists" error as unsuitable for Retry: https://github.com/ligato/vpp-agent/blob/master/plugins/kvscheduler/api/kv_descriptor_api.go#L212-L217). But for Contiv this is not relevant (and loopbacks are not used beyond that), so I guess we can also leave it as is and just to keep that scenario in mind.